frame/multisig/src/lib.rs

```diff
@@ -283,17 +281,8 @@ decl_module! {
 
 			let id = Self::multi_account_id(&signatories, 1);
 
-			// We're now executing as a freshly authenticated new account, so the previous call
-			// restrictions no longer apply.
-			let _guard = ClearFilterGuard::<T::IsCallable, <T as Trait>::Call>::new();
-			ensure!(T::IsCallable::filter(&call), Error::<T>::Uncallable);
```

Comment:
I have to admit that I don't understand why the `ClearFilterGuard` was added and now removed. Feel free to comment if you want feedback on it :-)

Reply:
@apopiak it is deprecated because of https://github.com/paritytech/substrate/pull/6318